code/modules/issue_reporter/issue_reporter.dm

```diff
@@ -0,0 +1,101 @@
+
+/datum/issue_reporter
+	var/client/owner
+
+/datum/issue_reporter/New(client/user)
+	if (!user)
+		terminate_reporter()
+		CRASH("Issue reporter created with no client.")
+	. = ..()
+	owner = user
+	ui_interact(user.mob)
+	// Clear references to the client
+	RegisterSignal(user, COMSIG_PARENT_QDELETING, PROC_REF(terminate_reporter))
+
+/datum/issue_reporter/Destroy(force, ...)
+	owner = null
+	return ..()
+
+/datum/issue_reporter/ui_state(mob/user)
+	return GLOB.always_state
+
+/datum/issue_reporter/ui_interact(mob/user, datum/tgui/ui)
+	ui = SStgui.try_update_ui(user, src, ui)
+	if(!ui)
+		ui = new(user, src, "IssueReporter")
+		ui.open()
+
+/datum/issue_reporter/ui_act(action, list/params, datum/tgui/ui, datum/ui_state/state)
+	. = ..()
+	// This could result in some horrible exploits if there is another exploit involved
+	if (usr.client != owner)
+		return
+	if (action != "submit")
+		return
+	var/githuburl = CONFIG_GET(string/githuburl)
+	if (!githuburl)
+		tgui_alert_async(usr, "The Github URL is not set in the server configuration.", "Configuration Error")
+		return
+	// This will open on the users own browser.
+	var/isRegression = params["isRegression"]
+	var/title = params["title"]
+	var/expected = params["expected"]
+	var/actual = params["actual"]
+	var/replicationSteps = params["replicationSteps"]
+	var/template_path = CONFIG_GET(string/issue_template)
+	var/body = rustg_file_read(template_path)
+	if (!body)
+		// /Default body
+		body = @{"## Occurance Details
+
+**Round Date:** %DATE%
+
+**Round ID:** %ROUNDID%
+
+**Testmerges**:
+%TESTMERGES%
+
+## Bug Details
+
+### Expected Behaviour:
+
+%EXPECTED%
+
+### Actual Behaviour:
+
+%ACTUAL%
+
+### Reproduction:
+
+%REPRODUCTION%
+"}
+	var/servername = CONFIG_GET(string/servername)
+	body = replacetext(body, "%DATE%", time2text(world.timeofday,"DD-MM-YYYY hh:mm:ss"))
```

Comment:
Please stop forcing this horrible date format on us. It makes no sense. YYYY-MM-DD is the international standard. [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601)

Reply:
This is the format specified in the github issue itself.

Reply:
Yes, and it's wrong. Change it

Reply:
This is the best format, but I updated it due to the newly added date format rules